he.json

```diff
@@ -0,0 +1,297 @@
+                "last-date" : "נראה לאחרונה:",
+                "occurrences" : "פעמים באייטם שופ: "
+            },
+            "reminder-count" : "משתמשים עם [תזכורות](/reminders):",
+            "add-reminder" : "הוסף תזכורת",
+            "remove-reminder" : "הסר תזכורת",
+            "has-reminders" : "הגדרת תזכורת לפריט זה.",
+            "no-reminders" : "תזכורות אינן זמינות עבור פריט זה.",
+            "images" : "תמונות",
+            "images-icon" : "אייקון",
+            "images-png" : "PNG",
+            "images-gallery" : "גלרייה",
+            "images-featured" : "בולט",
+            "interest-title" : "רמת עיניין לאורך זמן",
+            "interest-line1" : "גרף זה מייצג את רמת העיניין באייטם הזה לאורך זמן, כאשר המקסימום הוא הזמן הפופולארי ביותר",
+            "interest-line2" : "מידע לגבי היום הנוכחי עשוי להיות לא שלם",
+            "occurrences-table" : {
+                "title" : "הופעות בחנות",
+                "th-date" : "תאריך",
+                "th-days" : "מספר ימים לפני"
+            },
+            "set" : "סט",
+            "set-name" : "חלק מ[name]",
+            "popular-title" : "אייטמים פופולאריים היום"
+        },
+        "account" : {
+            "home" : {
+                "title" : "נתוני החשבון שלך",
+                "error" : "משהו השתבש בזמן טעינת נתוני החשבון שלך, אנא נסה שוב מאוחר יותר.",
+                "info" : {
+                    "title" : "מידע",
+                    "line1" : "אתה מחובר בתור [username] בעזרת [provider]",
+                    "line2-api-yes" : "יש לך[גישה לapi](/api/docs)",
+                    "line2-api-no" : "אין לך גישה לapi."
+                },
+                "migration" : {
+                    "title" : "חשבונות קודמים",
+                    "waiting-line1" : "עברנו להשתמש בדיסקורד לצורך התחברות.",
+                    "waiting-line2" : "אם יש לך חשבון עוד מ2018, תצטרך להתחבר אליו ולשייך אותו לחשבון הזה",
+                    "twitter-line1" : "עברנו להשתמש בדיסקורד לצורך התחברות. אתה כרגע משתמש בחשבון מדור קודם שעלול להיפסק בעתיד ושכבר עכשיו בעל פונקציונליות מוגבלת.",
+                    "twitter-line2" : "כדי לעבור לחשבון דיסקורד, לחץ על הכפתור מטה"
+                },
+                "ratings" : {
+                    "title" : "דירוגים",
+                    "none" : "אין לך דירוגים."
+                },
+                "destroy" : {
+                    "title" : "השמד חשבון",
+                    "line1" : "אם אינך רוצה עוד את חשבונך, תוכל למחוק אותו ואת כל הנתונים שלו [כאן](/account/destroy)"
+                }
+            },
+            "oauth" : {
+                "title" : "התחבר לfnbr.co",
+                "discord-button" : "התחבר בעזרת דיסקורד",
+                "legacy-text" : "לחלופין, אם יש לך חשבון דור קודם משנת 2018 הזקוק להעברה:",
+                "twitter-button" : "התחבר בעזרת טוויטר",
+                "privacy-text" : "על ידי כניסה אתה מסכים לנו לאחסן נתונים המשויכים לחשבונך כמפורט [במדיניות הפרטיות](/privacy) שלנו."
```

Comment:
It's not allowed to separate the link name (within the square brackets `[...]`) from the link url (inside the round brackets `(...)`). It must be in this format: `[link name](link url)`